### DISCUSSION TITLE
After posting a message I received a weird "Run failed at startup" mail

### DISCUSSION THREAD

**Discussion by xerces8** [community/community#59528] [Discussions]
### Select Topic Area

General

### Body

I posted a new discussion here (link below) and after a few seconds I received an email from notifications-at-github.com with the subject:

`[community/community] Run failed at startup: issue and project labeler - main (1feab22)`

And body:

```
[community/community] issue and project labeler workflow run
 
issue and project labeler: No jobs were run
 
View workflow run
 
You are receiving this because you are subscribed to this thread.
```

Is this some internal error on github?

**Comment by xerces8** [comment]
I received an almost exactly same mail after posting the above too.

**Comment by ris58h** [comment]
I wanted to report a bug with Releases via new discussion but got the same workflow error. My bugreport discussion wasn't created BTW.
UPDATE: it was created but can't be found by `author` filter.

**Reply by martylamoureux** [reply]
I also have a bug with Releases (drafts doesn't show up) and also posted a discussion which is also not visible.

You have the same issue of draft not accessible?

**Reply by ris58h** [reply]
@martylamoureux yes, you aren't alone - we are facing the same issue https://github.com/orgs/community/discussions/59535

**Comment by ris58h** [comment]
There a lot of failed workflows https://github.com/community/community/actions/workflows/issue_projects_labeler.yml
